repos/spack_repo/builtin/packages/seacas/package.py

```diff
@@ -334,6 +337,9 @@ class Seacas(CMakePackage):
         when="@:2023-10-24",
     )
 
+    patch("2022_ioss_win_defines.patch", when="@:2022-10-14")
```

Comment:
windows only patches?

Reply:
Yes, they're just preliminary at this point though, this almost certainly needs a lower bound, and presumably there's some upstream patch that also does this, I'm just trying to get _a_ passing build before cleaning up.